### DISCUSSION TITLE
Support loading environment variables from .env files (dotenv)

### DISCUSSION THREAD

**Discussion by exoaturner** [helmfile/helmfile#1719] [Ideas]
There was an open feature request for supporting .env files in the previous repository but was never moved across.

> The support for requiredEnv is really handy but it would be even more so if environment variables found in .env files was included.
> 
> Docker compose has fairly advanced support for .env files. But less could do it. Simply including variables found in .env would be a good start. Please see the below link for more information.
> 
> https://docs.docker.com/compose/environment-variables/

Link to original feature request: https://github.com/roboll/helmfile/issues/1461

**Comment by yxxhero** [comment]
@exoaturner I saw that it already had an PR for this feature. so we can move it in helmfile/helmfile repo.
